src/components/header/components/mobile/Mobile.component.tsx

```diff
@@ -0,0 +1,28 @@
+import { CartComponent } from '@/components/header/components/shared/Cart.component.tsx';
+import { LogoComponent } from '@/components/header/components/shared/Logo.component.tsx';
+
+import styles from './mobile.module.css';
+
+export const MobileComponent = () => (
+    <header className={styles.header}>
+        <LogoComponent />
+        <ul className={styles.menu}>
+            <li>
+                <CartComponent />
+            </li>
+            <li>
+                <a href={'#'}>
```

Comment:
need to fix linter. Change to `href="/"`

Reply:
done